### PR TITLE
[QRadar] Fallback for event columns

### PR DESCRIPTION
**Pull request by ilaner**
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-28972?filter=-1

## Description
Added fallback to default event columns in the query to search events. 

If the user set an empty field for some reason, we can override it with the default value.

### CODE REVIEW

File: Packs/QRadar/ReleaseNotes/2_4_38.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IBM QRadar v3
+
+- Added fallback to default event columns in the query to search events.
```
Comment (tkatzir) [inline Packs/QRadar/ReleaseNotes/2_4_38.md:6]: @ShirleyDenkberg , please review.
```suggestion
- Added fallback to default event columns in the query to search events, in the following commands: **qradar-searches**, **qradar-search-create**, and **qradar-search-retrieve-events**. 
```

**Review by tkatzir — APPROVED** [review]
Good!
Approved, with a comment to review the English by @ShirleyDenkberg .

**Comment by ShirleyDenkberg** [comment]
@tkatzir Doc review completed.

**Comment by ilaner** [comment]
Needs force merge as there is an issue with polling on XSOAR server master

**Pull request merged**